generators/app/templates/.eslintrc

```diff
@@ -0,0 +1,9 @@
+---
+parser: babel-eslint
+
+env:
+  es6: true
+  amd: true
+  node: true
+  mocha: true
+  browser: true
```

Comment:
I could easily pull in the entire file from the kibana repo then. I thought there was a default set of rules that it shipped with though... maybe that was just jslint though.